docs/source/user_manual/search_params.rst

```diff
@@ -173,6 +173,26 @@ Search parameters are set extensively via the :py:attr:`~kbmod.run_search.run_se
 | ``v_arr``              | [92.0, 526.0, 256]          | Minimum, maximum and number of         |
 |                        |                             | velocities to search through.          |
 +------------------------+-----------------------------+----------------------------------------+
+| x_pixel_bounds         | None                        | A length two list giving the starting  |
+|                        |                             | and ending x  pixels to use for the    |
+|                        |                             | search. `None` uses the image bounds.  |
++------------------------+-----------------------------+----------------------------------------+
+| x_pixel_buffer         | None                        | An integer length of pixels outside    |
+|                        |                             | the image bounds to use for starting   |
+|                        |                             | coordinates. If `x_bounds` is provided |
+|                        |                             | that takes precedence.                 |
+|                        |                             | `None` uses the image bounds.          |
++------------------------+-----------------------------+----------------------------------------+
+| y_pixel_bounds         | None                        | A length two list giving the starting  |
+|                        |                             | and ending y pixels to use for the     |
+|                        |                             | search. `None` uses the image bounds.  |
++------------------------+-----------------------------+----------------------------------------+
+| y_pixel_buffer         | None                        | An integer length of pixels outside    |
```

Comment:
A thing I recently learned:

* double backtics render highlighted text, f.e. ` ``key`` ` --> `key`
* single backtics render italicized text, f.e, `key` --> _key_
* it's probably a good idea to render key names in highlight - f.e. ` ``x_pixel_bound`` ` --> `x_pixel_bound`
* it's probably a good idea to render key values in highlight with quotations - f.e. ` ``"a_string_value"`` ` --> `"a_string_value"` 

to produce the most consistent documentation.